common/test/views/support/cleaner/AtomCleanerTest.scala

```diff
@@ -32,7 +32,7 @@ class AtomCleanerTest extends FlatSpec with Matchers with FakeRequests {
 
 
  private def clean(document: Document, atom:Option[Atoms], amp: Boolean): Document = {
-    val cleaner = AtomsCleaner(youTubeAtom, shouldFence = false, amp = amp)(TestRequest())
+    val cleaner = AtomsCleaner(youTubeAtom, amp = amp)(TestRequest())
```

Comment:
Before your change `shouldFence` would be `false`. Now it would be `true` because of default value. Does this matter in this test?